tests/app/delivery/test_send_to_providers.py

```diff
@@ -274,7 +274,7 @@ def test_should_send_sms_with_downgraded_content(notify_db_session, mocker):
     mocker.patch("app.aws_sns_client.send_sms")
 
     mock_phone = mocker.patch("app.delivery.send_to_providers.get_phone_number_from_s3")
-    mock_phone.return_value = "15555555555"
+    mock_phone.return_value = "+12028675309"
```

Comment:
Just a quick note here, should be keeping the `15555555555` number for testing purposes instead of changing back to a potentially in-use number?

Reply:
No.  As part of adding the simulated check before we send the message, we have to validate the phone number format and the validator detects that '555' is not in use.   The test code is already full of this phone number, there must be 100 uses of it and several close relatives.   Figuring out what to do here and maybe conditionally disabling the validator based on test environment (??? doesn't seem like a good idea but ...) is for another day.

Reply:
Ah gotcha, thanks!